Kitodo/src/test/java/org/kitodo/selenium/testframework/helper/WebDriverProvider.java

```diff
@@ -35,7 +35,7 @@
     // https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection
     // Please don't rely on the LATEST_RELEASE file without a version suffix.
     // It exists for backward compatibility only, and will be removed in the near future.
-    private static final String CHROME_DRIVER_LATEST_RELEASE_URL = "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_83";
+    private static final String CHROME_DRIVER_LATEST_RELEASE_URL = "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_85";
```

Comment:
Should this change not be moved into a separate pull request which is first merged before other pull requests get merged so that selenium execution is working again?